src/bundle/Resources/views/content/tabs/details.html.twig

```diff
@@ -0,0 +1,89 @@
+<div class="ez-list-toolbar">
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.title'|trans({}, 'locationview') }}</h2>
+</div>
+
+<table class="ez-table-data">
+    <thead>
+        <tr>
+            <th>{{ 'locationview.details.creator'|trans({}, 'locationview') }}</th>
+            <th>{{ 'locationview.details.lastcontributor'|trans({}, 'locationview') }}</th>
+            <th>{{ 'locationview.details.versions'|trans({}, 'locationview') }}</th>
```

Comment:
note that this translation should be updated in i18n as the label is currently incorrect